waterlog/src/Components/Wastage.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import DailyWastageComponent from './DailyWastage';
+import MonthlyWastageComponent from './MonthlyWastage';
+import SeasonalWastageComponent from './SeasonalWastage';
+
+class Wastage extends Component {
+    constructor(){
+      super();
+      this.state = {
+        display: "daily"
+      }
+    }
+    componentDidMount(){
+      document.getElementById("openByDefault").click();
```

Comment:
I have tried that but it doesn't recognize the 'e'

Reply:
![e](https://user-images.githubusercontent.com/46716201/52622582-d68afc80-2eb2-11e9-9fec-bf8e725f944f.PNG)

Reply:
Let me attempt to explain what I meant a bit more detailed.

Firstly accessing the document is never recommended. Why? Because if you ever need to implement server-side rendering then accessing the document can cause issues. And currently server-side rendering is often still needed for SEO (Search Engine Optimization) purposes.
Always first try to see if you can solve your issue using react as it's designed before accessing the document.

Here is how I would solve this issue

Updates to the render method: Use variable to add/remove `active` class on the buttons
```
<button className={`btn-graph-nav tablinks ${this.state.display === "daily" ? "active" : ""}`} // note here: using variables to add/remove className to prevent accessing the document
	onClick={(e)=>this.openGraph(e,"daily")}
	id="openByDefault">
	Daily
</button>
<button className={`btn-graph-nav tablinks ${this.state.display === "monthly" ? "active" : ""}`} // and here
	onClick={(e)=>this.openGraph(e,"monthly")}>
	Monthly
</button>
<button className={`btn-graph-nav tablinks ${this.state.display === "seasonal" ? "active" : ""}`} // and here
	onClick={(e)=>this.openGraph(e, "seasonal")}>
	Seasonal
</button>
```

Now the `openGraph` function no longer needs the event passed in so that can be updated to `openGraph = (cityName) => {`

Now you can update `componentWillMount` to
```
componentWillMount(){
    this.openGraph("daily");
}
```

Consider updating the styling for `tabcontent` and `tablinks` in the same manner

Reply:
Also maybe readup on refs for another proper way of accessing elements in react https://reactjs.org/docs/refs-and-the-dom.html